src/main/webapp/i18n/en/exam.json

```
@@ -691,6 +691,10 @@
                 "actions": "Actions",
                 "viewResultsOrRunDetection": "View Results or Run Detection",
                 "viewCases": "View Cases"
+            },
+            "dateChange": {
+                "title": "Changing the working time of an active exam will disrupt the workflow of the students!",
```

```suggestion
                "title": "The exam conduction is currently in progress. Changing the times of an active exam will notify all active students about the duration change",
```

Change request from discussion w/ @krusche